tests/ios/unit-test/test-imports-app/BUILD.GoogleMobileAds

```diff
@@ -0,0 +1,36 @@
+
+load(
+    "@build_bazel_rules_ios//rules:apple_patched.bzl",
+    "apple_static_framework_import"
+)
+
+apple_static_framework_import(
+    name = "GoogleMobileAds",
+    framework_imports = glob(["Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.xcframework/ios-arm64_x86_64-simulator/GoogleMobileAds.framework/**"], allow_empty = False),
+    sdk_frameworks = [
+        "AudioToolbox",
+        "AVFoundation",
+        "CFNetwork",
+        "CoreGraphics",
+        "CoreMedia",
+        "CoreTelephony",
+        "CoreVideo",
+        "MediaPlayer",
+        "MessageUI",
+        "MobileCoreServices",
+        "QuartzCore",
+        "Security",
+        "StoreKit",
+        "SystemConfiguration"
+    ],
+    weak_sdk_frameworks = [
+        "AdSupport",
+        "JavaScriptCore",
+        "SafariServices",
+        "WebKit"
+    ],
+    sdk_dylibs = [
+        "libsqlite3",
+        "libz"
+    ],
```

Comment:
This needs a visibility modifier of public it looks like:

````
ERROR: /Users/jmarino/Development/rules_ios/tests/ios/unit-test/test-imports-app/BUILD.bazel:14:16: in swift_library rule //tests/ios/unit-test/test-imports-app:TestImports-App_swift: target '@GoogleMobileAdsSDK//:GoogleMobileAds' is not visible from target '//tests/ios/unit-test/test-imports-app:TestImports-App_swift'. Check the visibility declaration of the former target if you think the dependency is legitimate
ERROR: Analysis of target '//tests/ios/unit-test/test-imports-app:TestImports-Unit-Tests' failed; build aborted: Analysis of target '//tests/ios/unit-test/test-imports-app:TestImports-App_swift' failed
```

Reply:
Awesome I am on it